test/app.ts

```diff
@@ -2,13 +2,32 @@ import fastify, { FastifyPluginAsync } from 'fastify';
 import { ItemService } from 'graasp';
 import { TaskRunner } from 'graasp-test';
 import common from './common';
+import { GraaspPluginValidationOptions } from '../src/types';
+import { ServiceMethod } from 'graasp-plugin-file';
 
 type props = {
   runner: TaskRunner;
-  plugin: FastifyPluginAsync;
+  options?: GraaspPluginValidationOptions;
+  plugin: FastifyPluginAsync<GraaspPluginValidationOptions>;
   itemService: ItemService;
 };
 
+export const DEFAULT_OPTIONS = {
+  classifierApi: 'localhost',
+  serviceMethod: ServiceMethod.LOCAL,
+  serviceOptions: {
+    s3: {
+      s3Region: 's3Region',
+      s3Bucket: 's3Bucket',
+      s3AccessKeyId: 's3AccessKeyId',
+      s3SecretAccessKey: 's3SecretAccessKey',
+    },
+    local: {
+      storageRootPath: 'storageRootPath',
+    },
```

Comment:
I think you used this part in another place in tests. Could you refactor and reuse it in both places?